app/onboarding/page.tsx

```diff
@@ -52,11 +53,14 @@ export default function Onboarding() {
             </UpdateText>
           </UpdateContainer>
         </Container>
-        <ButtonContainer>
-          <ContinueButton>
```

Comment:
for some reason, when i click on the continue button, there is a red underline? can you try fixing this